src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
```

Comment:
Bad naming. Should not specify that it`s "ballNumber" as it is already obvious from context (variable number of a Ball class)